src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 12px;
+  width: 12px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin: 2px 6px 2px 2px;
```

Comment:
don't use vertical margin for the stars, just margin-right
center background-image